models/modules/attention_layers.py

```diff
@@ -0,0 +1,59 @@
+import torch
+import einops
+import torch.nn as nn
+
+
+class MultiHeadAttention(nn.Module):
+    def __init__(self, channels: int, heads: int = 4):
+        """
+        Multi Head Attention Layer
+
+        Args:
+            embedding_size (int): size of embedding dim
+            heads (int): number of attention heads
+        """
```

Comment:
I'm not sure what's the relation between channels and embeddings? 

As far as I can tell, the number of channels is used for this argument, so in the middle where the shape is `[B, 128, 250]`, `embedding_size` is set to 128, which could suggest that we have 250 embeddings each with dim=128, and I'm not sure if this description makes sense. Do you know if there's a good reason to use the name `embedding_size` here?

Reply:
Oh, I forgot to change that. I'll update it, it should be called channels